src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,64 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.nav__list {
+  display: flex;
+  background-color: white;
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__item {
+  display: flex;
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc ;
+  border-radius: 8px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  cursor: pointer;
+}
```

Comment:
[BEM] - create a separate file per each BEM block styles that have the same name as the block
nav is separate BEM block